src/components/GisidaLite/tests/index.test.tsx

```diff
@@ -10,7 +10,7 @@ describe('components/GisidaLite/arePropsEqual', () => {
   it('returns false if layers length has changed', () => {
     const prevProps: GisidaLiteProps = {
       layers: [],
-      mapCenter: undefined,
+      mapCenter: [10, 10],
```

Comment:
Does this and line 38 below mean that the map center is by default defined to [10, 10]?  Why?

Reply:
I added that value for testing purposes

Reply:
Cool